\documentclass[preprint,aps,draft]{revtex4}

\begin{document}

\title{Modeling chromosomes during meiosis in fission yeast}% using pinned polymer rings}

\author{Wenwen Huang}
\affiliation{Max Planck Institute for the Physics of Complex Systems, 01187, Dresden, Germany}

\author{Yen Ting Lin}
\affiliation{Max Planck Institute for the Physics of Complex Systems, 01187, Dresden, Germany}

\author{Daniela Fr\"{o}mberg}
\affiliation{Max Planck Institute for the Physics of Complex Systems, 01187, Dresden, Germany}

\author{Petrina Delivani}
\affiliation{Max Planck Institute of Molecular Cell Biology and Genetics, 01307, Dresden, Germany}

\author{Mariola Chac\'{o}n}
\affiliation{Max Planck Institute of Molecular Cell Biology and Genetics, 01307, Dresden, Germany}

\author{Iva Tolic} 
\affiliation{Max Planck Institute of Molecular Cell Biology and Genetics, 01307, Dresden, Germany}

\author{Frank J\"{u}licher}
\affiliation{Max Planck Institute for the Physics of Complex Systems, 01187, Dresden, Germany}

\author{Vasily Zaburdaev}
\affiliation{Max Planck Institute for the Physics of Complex Systems, 01187, Dresden, Germany}

\date{\today}

\begin{abstract}
%Ring structured molecular has been discovered are ubiquitous among biological processes and play an %important role, e.g., circular RNA controlling transcription. 
During the prophase of meiosis in fission yeast, both ends of chromasomes are bonded to the spindle pole body (SPB) and form a ring structure. Furthermore, the whole nucleus oscillates moving from one pole of the elongated cell to the other. Oscillations are driven by the dynein motors and microtubules which are attached to the SPB. The dramatic movements of the nucleous are believed to promote the chromatin alignment as necessary for recombination. Our goal is to understand the physical picture of chromosome alignment during nuclear oscillations. 
We perform extensive Brownian dynamics simulations of three pairs of homologous chromosomes during the oscillations. Indvidual chromosome is represented by a bead-rod ring, where the SPB is a special common bead shared by all the rings. A periodic force is applied to the SPB, which pulls the chromosomes through the viscous nucleoplasm and under the confinment of the cell walls. We use the available data on the persistence length, size and the compation ratio of the chromosomes to setup the parameters of our simulations. We analyze the distance between the homologous loci as the function of time and amplitude of oscillations and compare it to the experimental data obtained for the fission yeast. Our results provide a quantitative characterization of the role of oscillations on the alignment of chromosomes during meiosis.

%To solve the dynamical equations under the constraints of fixed rod length, a predictor-corrector algorithm is applied. More generally, the numerical scheme we developed here can applied to bead-rod system with any kinds of topological constraints.
%The distance between every two beads, which are directly related to to biological processes such as paring and recombination, are measured in both the equilibrium case of steady moving velocity and non-equilibrium oscillating case.
%Furthermore, we studied the influence of other realistic interactions to the statistics of distance distribution, e.g. Repulsive force counts for the excluded volume effect.
%Our study gives an general numerical framework to study the dynamics of bead-rod polymer system and helps us to understand the exact role of nuclear oscillation in meiosis of fission yeast.
% Here please put the description of your project as in the larger context: namely detailed 3 dimensional simulations of the chromosome oscillation dynamics, and not the content of the paper. About the paper I will try to give a talk. So no reason to repaet ourselves.

%Using the pinned polymer ring model, we investigate the statistics of the loci position along chromatins.
%Based on the concept of Brownian bridge, we derived the analytical result of distance between homologous pairs under certain noise level.
%Furthermore, we verified our theoretical results using the Brownian dynamics simulation based on a bead-rod polymer model.
%Our analytically solvable model helps us to understand the exact role the chromosome movements and is also applicable to other ring structured biological and soft mater cases.

	

\end{abstract}
\maketitle

\end{document}

