%*******************************************************************************
%*********************************** First Chapter *****************************
%*******************************************************************************

\chapter{Introduction}  %Title of the First Chapter
\graphicspath{{Chapter1/Figs/}}


Many biological processes can be transferred to well-studied models in statistical physics. Examples are ranging from the random walk model of mRNA during the transcription \cite{Macdonald1968,Bressloff2013,Chou2011} to the evolution of ecological systems \cite{Berryman1992,Butler2009,Sugihara2012,Williams2016a}. Analysis of these extracted models could help us to get useful insights into the corresponding biological problems. In this thesis, we are interested in modeling a molecular level biological problem, i.e. the motion of chromosomes in fission yeast during meiotic cell division \cite{Chikashige1994,Vogel2009,Shibuya2014}.

Interestingly, during Prophase I of meiotic fission yeast, the nucleus containing three pairs of chromosomes move from one pole of the rod-like cell to the other, forming an oscillation behavior last for about two hours. The period of the oscillation is about $10$ minutes. Cells are divided into two after the oscillation is done \cite{Yamamoto2001,Ding2004,Gerton2005}.

We are going to model this oscillation processes quantitatively with the chromosomes represented by polymer models \cite{DeGennes1979,Doi1988}. The results we obtained from the modeling are used to explain relevant biological functionalities like chromosome alignment and gene recombinations \cite{Lin2015}. 

In this chapter, we will introduce the biological backgrounds and some previous related studies about modeling chromosomes with polymers. We will propose our research goals and give an overview of the thesis in the last section. 


%********************************** %First Section  **************************************
\section{Nuclear oscillation in fission yeast} %Section - 1.2
\label{sec:nuclear_oscillation_in_fission_yeast}

In this section, we will introduce the biological basis of nuclear oscillation in fission yeast. We will firstly introduce our model organism, fission yeast. And then we show some basis of meiosis cell division. Next, we go into the nuclear structure of fission yeast and the movements of chromosomes, specifically during meiosis. The biological processes like chromosome alignment and recombination are discussed. In the last subsection, we discuss some understandings for the biological role of the nuclear oscillation.

\subsection{Fission yeast}
\label{sub:fission_yeast}

Fission yeast, also named \emph{Schizosaccharomyces pombe}, is a model organism that widely used in the study of molecular and cell biology. It is a unicellular eukaryote and has a rod-like shape. Typical size of fission yeast is $3$-$4$ micrometres in diameter and $7$-$14$ micrometres in length \cite{Forsburg2003,Fantes2016}. See in Fig. \ref{fig:fissionYeast} for a illustration of fission yeast.

\begin{figure}[htpb]
    \centering
    \includegraphics[width=0.5\linewidth]{fissionYeast}
    \caption{Microscopic view of a fission yeast culture. The scalar bar indicates $10$ $\mu$m. Image reprinted from \cite{Morgan2007} with permission. }
    \label{fig:fissionYeast}
\end{figure}

Fission Yeast is widely used in traditional brewing and baking. It was first discovered in 1893 in the sediment of millet beer \cite{Fantes2016,Hoffman2015}. As a single-celled fungus, fission yeast has a simple genome with three different chromosomes. The genome of fission yeast is fully sequenced and the three chromosomes contain about $14$Mb of DNA \cite{Wood2002}. It has a rapid growth rate and easily manipulated to make mutants, which make it a perfect modeling organism for genetic studies. The growth of the fission yeast is simply by the elongation at the ends. After mitosis, division occurs by the formation a cell plate that cleaves the cell at its midpoint \cite{Forsburg2003}. 

Fission yeast is normally a haploid cell. However, when put under stressful conditions, such as nitrogen deficiency, two cells will conjugate to form a diploid and then form four spores via meiosis \cite{Coelho2013}. This is easy to observe experimentally and this stage is exactly when the interesting nuclear oscillation happens \cite{Chikashige1994}. In the next subsection, we will explain the basis of the meiosis in fission yeast. 



\subsection{Basis of meiosis}
\label{sub:basis_of_meiosis}

Meiosis is a kind of cell division that reduces the number of chromosomes in the parent cell by half and produces four genetically distinct gamete cells. This process occurs in all the sexually reproducing organisms, including human \cite{Freeman2008}. 

Meiosis begins with a parent cell with two copies of each chromosome, and is followed by two rounds of cell divisions which produce four potential daughter cells, each has half number of chromosomes as their parent cell. The two rounds of cell division are called \emph{Meiosis I} and \emph{Meiosis II}, respectively. It is during Meiosis I that the pair of chromosomes, one from the father and the other from the mother, separates into two offspring cells. Meiosis II is very similar to the mitosis where two sister chromatins separate \cite{Freeman2008,Villeneuve2001a}. See an overview of meiosis in Fig. \ref{fig:meiosis}.

\begin{figure}[htpb]
    \centering
    \includegraphics[width=0.8\linewidth]{meiosis}
    \caption{Overview of meiosis and an illustration of recombination between homologous chromosomes resulting in four unique daughter cells. Image reprinted from \cite{} with permission.}
    \label{fig:meiosis}
\end{figure}

As in mitosis, each round of cell division can be divided in prophase, metaphase, anaphase, and telophase. We will elaborate Meiosis I in details, especially the Prophase I when the nuclear oscillation happens. 

$\bullet$ \emph{Prophase I}: prophase I is an important stage that many processes happened. Two of the examples are bouquet formation \cite{Wegener1980a} and homologous recombination \cite{Davis2001,Gerton2005}, both occurring in generic organisms. In the early state of Prophase, chromosomes are reorganized spatially, usually, the telomeres are clustered and attached to a small region of the nuclear membrane, forming a bouquet structure. This is called bouquet formation or telomere clustering in biology \cite{Chikashige1994,Wegener1980a,Niwa2000}, see in Fig \ref{fig:prophase} for an example in fission yeast. In the process of recombination, the homologous chromosomes, which are paternal and maternal pairs, align and exchange some parts of their DNA and usually results in the chromosomal crossover. Homologous recombination is critical for pairing and accurate segregation of the chromosomes in the later stage of Meiosis I. More interestingly, this stage is exact the period when the nuclear oscillation happens in fission yeast \cite{Ding1998,Wells2006}. We will devote this part to next subsection.

\begin{figure}[htpb]
    \centering
    \includegraphics[width=0.8\linewidth]{prophase}
    \caption{The meiotic prophase I of fission yeast. Telomeres are clustered to form a bouquet structure. Image modified and reprinted from \cite{Asakawa2007}.}
    \label{fig:prophase}
\end{figure}

$\bullet$ \emph{Metaphase I}: in this stage, homologous pairs move together along the middle plate, the microtubules from centrosomes attach to their respective chromosomes, the paired homologous chromosomes align along an equatorial plane that bisects the spindle. However, in Fission Yeast, the centromere is replaced by a functional equivalent organelle called spindle pole body (SPB) \cite{Freeman2008}.

$\bullet$ \emph{Anaphase I}: in this stage, the microtubules shorten, pulling homologous chromosomes to opposite poles. Notice here, chromosomes still consist of a pair of sister chromatids. The cell body elongates, preparing for cell division \cite{Freeman2008}.

$\bullet$ \emph{Telophase I}: in the last stage of Meiosis I, chromosomes arrive at the poles. The microtubules network of spindle disappears. New nuclear membrane appears. The two daughter cells now only have half the number of chromosomes \cite{Freeman2008}.

After Meiosis I, Meiosis II occurs without DNA replication in between. The process is similar to Meiosis I except the sister chromatids segregate instead of homologous chromosomes \cite{Freeman2008}. Four unique daughter cells are formed after the completion of Meiosis. The homologous recombination process takes an important role for this uniqueness.

\subsection{Nuclear oscillation}
\label{sub:nuclear_oscillation}

As mentioned in the previous subsection, nuclear oscillation happens during prophase I of meiosis in fission yeast, and so as the important processes of chromosomes homologous alignment and recombination \cite{Ding1998}. Because the impressive shape of nucleus during this stage, nuclear oscillation also often mentioned as \emph{horse-tail} movements in biology\cite{Koszul2009a,Wells2006,Davis2001}, see in Fig. \ref{fig:oscillation} for a time-lapse illustration. We believe the chromosome movements play an important role in this process and decide to model it quantitatively. In this subsection, we are going to elaborate the details of nuclear oscillation. We will answer the questions like what is the internal structure of nucleus during oscillation, what is the driven force of the oscillation, how long the oscillation lasts and what is the time period of the oscillation, etc.
\begin{figure}[htpb]
    \centering
    \includegraphics[width=1.0\linewidth]{oscillation}
    \caption{Time-lapse experiments of nuclear oscillation in fission yeast, DNA marker in green (Rec8-GFP) and SPB marker in magenta (Sid4-mCherry) also indicated by asterisk. Reprinted from \cite{Chacon2016} with permission.}
    \label{fig:oscillation}
\end{figure}

\subsubsection{The looping structure of chromosomes}
Before the nuclear oscillation, chromosomes are reorganized and the bouquet formation happens. In fission yeast, the SPB is anchored in the nuclear envelope, telomeres of chromosomes are clustered to the SPB region of the inner nuclear membrane. The chromosomes are condensed to be the rod-like chain. Notice that there are still two sister chromatins contained in one chromosome. With all the telomeres bond to the SPB, chromosomes form the looping structure, as we can see in Fig. \ref{fig:prophase}. 

\subsubsection{Redistribution of dynein motors drives the nuclear oscillation}
While the inner side of SPB bonds the chromosome telomeres, the outer side is attached to the microtubules in the cytoplasm. During the oscillation, dynein motors are the energy supplier. Interestingly, as one motor is not enough for the oscillation, the collective behavior of motors is observed to drive the nucleus motion. The spatial distribution of motor molecular varies during the oscillation. Motors accumulates in the side of fission yeast that the nucleus moves toward to. It is found that the pulling force is the main contribution that drives the oscillation \cite{Vogel2009}.

\subsubsection{Related biological parameters of nuclear oscillation}
To study the dynamics during oscillation, several parameters are estimated experimentally and input into our model later. These parameters are summarized in table \ref{tab:parameters}.

\begin{table}[htpb]
    \centering
    \caption{Parameters of fission yeast during meiosis}
    \label{tab:parameters}
    \begin{tabular}{l|l}
        \hline
        \textbf{Parameter} & \textbf{Value} \\
        \hline
        Typical size of nucleus          &  $3\mu$m \\
        Chromosome number                &  Three pairs \\
        Compaction ratio of chromatin    &  $10^2$bp/nm \\
        Kuhn length of chromatin         &  $100\sim300$nm  \\
        Duration of nuclear oscillation  &  $2$ hours  \\
        Period of nuclear oscillation    &  $10$ min  \\
        Moving speed of nucleus          &  $2.5\mu$m/min \\
        Viscosity of nucleoplasm         &  $1000\times \mu_{\text{water}}$ \\
        \hline
    \end{tabular}
\end{table}


\subsection{The role of nuclear oscillation}
\label{sub:the_role_of_nuclear_oscillation}

Although we can clearly observe the nuclear oscillation in fission yeast, the biological role of it is not thoroughly understood. One intuitive hypothesize is that the movement facilitates the paring of homologous \cite{Ding2004}. However, Koszul et al. proposed that the chromosome movement might play other roles than paring, such as resolve homologous entanglements or non-homologous connections \cite{Koszul2009a}. Also, Mariola et al. stated a dual role for the nuclear oscillation, promoting initial paring and restricting the time of chromosome associations to ensure proper segregation \cite{Chacon2016}. 

We believe nuclear oscillation plays an important role for the chromosome alignment. However, it is hard to image the exact mechanism of the alignment without going deeper of the process. That is why we propose a quantitative model in this thesis and study the statistical and dynamical details of the model, trying to understand to the machinery of paring quantitatively. 


%********************************** %Second Section  *************************************
\section{Chromosomes modeled as the polymer}
To quantitatively describe the chromosome, it is natural to model it as a polymer. In fact, there are already a lot of excellent examples in this direction \cite{Wong2013,Tree2013,Halverson2014,Jun2010a}. However, depending on the situations under considering, different polymer models may applied. 

In physics, a polymer model is usually described by beads connected by massless springs or rods. The interactions, usually characterized as different types of potentials, specify the setting of the model \cite{Doi1988}. As we want to model the chromosome during nuclear oscillation in fission yeast, there are two major factors we need to take into account besides some other minor details. First, the topology of the chromosome is a ring structure as shown in Fig \ref{fig:prophase}. Second, all chromosomes are bound to the SPB and pulled by an external force. According to these biological factors and the experimental measurements like Fig. \ref{fig:prophase}, we propose a pulled polymer loop model for the chromosomes in this specific situation. See in Fig. \ref{fig:schematic} for a sketch of our model. We will leave the discussion of the model details in afterward chapters.

\begin{figure}[htpb]
    \centering
    \includegraphics[width=0.8\linewidth]{schematic}
    \caption{The sketch of our pulled polymer loops model for chromosomes in meiotic fission yeast. Three pairs of chromosomes with all ends bound to SPB (shown in magenta) in the nucleus are indicated by different colors.  The SPB is pulled by multiple dynein motors (not shown) walking along microtubules (dark green). The SPB is anchored to the nuclear envelope (light green) and entrains the whole nucleus.}
    \label{fig:schematic}
\end{figure}

To the best of our knowledge, there are not many works which are discussing on not only the polymer loops but also the polymer is pulled by an external force. In the following subsections, we will introduce two related aspects of previous works, i.e. works about the polymer loops and works related to pulled polymer. 

\subsection{The ring polymer model}
\label{sub:the_ring_polymer_model}

Polymers forming a ring structure are ubiquitous in chemistry and biology \cite{Halverson2014,Richter2015}. The study of ring polymer can go back to the years when polymer physics was built up \cite{Kramers1946,Zimm1949}. Kramers developed an equilibrium theory that possesses branch points and rings of the dilute polymer solution in 1946 \cite{Kramers1946}. Zimm calculated the statistics of mean square radii of molecules containing branches and rings in 1949 \cite{Zimm1949}. However, comparing to the simplest polymer chain model, research focus on polymer ring are far less than the former. We will review here some interesting and most related ones. Of course, it is not possible to exhaust all related works here, we pay our attention particularly to those that are related to the chromosome modeling.

Since Kramers and Zimm, there are a number of works trying to study the static and dynamical properties of the ring polymer from the theoretical point of view.
In 1965, Casassa derived some statistical properties of flexible ring polymers, including mean square radius, the second Virial coefficient and angular distribution of scattering \cite{Casassa1965}. 1980, Burchard and Schmidt calculated the static and dynamical structure factors of flexible ring polymers \cite{Burchard1980}. Baumg\"{a}rtner considered the self-avoiding effect of ring polymers in 1984 and found the asymptotic scaling exponents are the same as linear polymers \cite{Baumgartner1982a}. In 1986, Cates et al. studied the nor-concatenated ring polymer melt and found the size of polymer $R$ scales with the polymerization index $N$ as $R\sim N^{2/5}$ and the diffusion constant $D$ scales as $D\sim N^{-2}$ \cite{Cates1986}.  In 1994, Obukhov et al. considered the dynamics of a ring polymer in a gel and obtained the diffusion coefficient scales with the molecular weight as $D\sim M^{-2}$ and the longest relaxation time $T$ scales as $T\sim M^{5/2}$ \cite{Obukhov1994}. Carl investigated the configurational and rheological properties of multiple-twisted ring polymers using a long cyclic finitely extensible bead-spring model in 1995 \cite{Carl1995}. He also presented a study on bead-spring chains in steady flows, various properties such like the power spectrum, the autocorrelation functions of configurational quantities were discussed in 1996 \cite{Carl1996}.
In 2001, Panyukov and Rabin studied the effects of thermal fluctuations on elastic rings. Analytical expressions are derived for some static and dynamical quantities \cite{Panyukov2001}.  Mukherji et al. studied a polymer ring or chain diffused around attractive surfaces. They found the diffusion constant scales as $D \sim N^{-3/2}$ linear chain and solid strong adsorbed surfaces, and $D\sim N^{-1}$ for ring polymer and soft surfaces in 2008 \cite{Mukherji2008}. Sakaue proposed a simple mean-field theory for the structure of ring polymer melts which takes into account the many-body effects \cite{Sakaue2011,Sakaue2012}. In 2012, Kim et al. presented a self-consistent field theory formalism of topologically unconstrained ring polymers \cite{Kim2012}.  In 2013, Reigh performed lattice Monte-Carlo simulations to investigate the dependence of ring polymer conformation to the concentration, where the scaling of gyration radius with the concentration $R_g \sim \phi^{-0.59}$ was found \cite{Reigh2013}. In 2014, Lang et al. studied the tumbling dynamics of semiflexible ring polymers as a model of the cytoskeletal filament in a shear flow. They found the tumbling frequency scales $f_c$ scales with the Weissenberg number as $f_c \sim Wi^{3/4}$ rather than the prediction of classical theory that $f_c \sim Wi^{2/3}$ \cite{Lang2014b}.

Ring polymer with high concentration, like polymer melts, are often used in modeling interphase chromosomes. Interestingly, it is found that during interphase, the spatial organization for the multiple chromosomes in the nucleus is not homogeneous and well mixed. Instead, each chromosome forms its own ``territories'' \cite{Halverson2014}. Many interesting works can be found respecting to this problem. For example, in 2008, Rosa and Everaers used the simulation results of polymers to explain the existence and stability of territories of interphase chromosomes in genetic eukaryotes \cite{Rosa2008}. Because usually, the computation power required to simulate the whole genome is huge, they also developed an efficient multiscale numerical approach to study the conformational statistics of ring polymers melts in \cite{Rosa2014b}.  Dorier employed a very simple non-permeable freely jointed polymer model and recovered the chromosomal territories in a crowded nuclei \cite{Dorier2009}. This part of work is well reviewed in \cite{Halverson2014}, the interested reader can refer to the references therein. 

On the other hand, it is also possible that ring polymer structures are formed temporarily in chromosomes. This could be caused by the DNA replication process or binding proteins connecting two loci of chromosomes. Many great works were done also in this direction. In 1995, Sachs use a looping random walk model to study to the interphase chromosomes, fluorescence labeled data is compared to the theoretical prediction \cite{Sachs1995}. Marko considered a model of two polymer rings tethered one another and its application to chromosome segregation in 2009 \cite{Marko2009}.  The looping probabilities of interphase chromosomes were also discussed in \cite{Rosa2010}.  In 2011, Zhang et al. modeled the meiotic chromosomes as a polymer that could form internal loops by binding proteins. They found the loops play an important role in the mechanical properties of the polymer \cite{Zhang2011a}. Wong set up a polymer model and use it to predict the whole nuclear architecture of fission yeast \cite{Wong2012}. Dekker and Giorgetti employed the computational polymer model to explain the 3C/HiC data \cite{Dekker2013,Giorgetti2014}. In 2014, Youngren employed ring polymer model to study the duplication and segregation of \emph{E. coli} chromosomes \cite{Youngren2014}.  These are just a few examples, more can be found if one is interested.

The confinement such as the nuclear membrane or cell shape could also take an important role in chromosome dynamics. One of the examples of this kind of work is Fritsche's work in 2011, they studied the influence of confinement geometry to the spatial organization of semiflexible ring polymers \cite{Fritsche2011}. The studies of the polymers (including chains and rings) under confinements were reviewed by Ha el al. in \cite{Ha2015}. 

There are also a lot of great experimental work related to ring polymers. In 1992, Tead et al. employed polystyrene molecules to perform experiments and compared the diffusion of linear and ring polymers \cite{Tead1992}. Kapnistos et al. found the stress relaxation of entangled ring polymer was power-law rather than exponential in \cite{Kapnistos2008}. Structure and dynamics of polymer rings by neutron scattering were studied by Br\'{a}s et al. in 2011. Witz et al. employed the atomic force microscopy to studied 2D circular DNA in \cite{Witz2008,Witz2011}.  Goo{\ss}en et al. studied dynamics of polymer rings using neutron spin echo spectroscopy which space-time evolution of segmental motion could be observed \cite{goossen2014,Goossen2015}. 

Due to its importance, the study of ring polymers is much intensive and causes much more attention nowadays. Besides what we have mentioned above, the shape of ring polymers is studied in \cite{Bishop1985,Jagodzinski1992,Alim2007,Reiss2011}. Also, there are a series of works considering the ring polymer with entanglements and topological knots \cite{Michels1982,Polymers1991,Koniaris1991,Grosberg1996,Shimamura2001,Orlandini2003,Tubiana2011,Uehara2014,Li2015a}.
We are only able to list a few of those great works. Interested readers can refer to the references therein. 

\subsection{Pulled polymer model}
\label{sub:pulled_polymer_model}

As we mentioned above, in order to model the nuclear oscillation of fission yeast, we have to consider the pulling dynamics. If we transfer the coordinate and sit on the pulled monomer, a pulled polymer is also equivalent to a pinned polymer in an external flow or force field. In this section, we would like to review some previous works in this direction. Most of them are about pulled polymer chain. Whatsoever, we think it is still helpful to know what have been done about pulled polymers or tethered polymer in an external field.

A polymer chain with one end free and the other end pulled by an external force was first discussed by de Gennes \cite{DeGennes1974,DeGennes1979}. After that, another important progress was made by Pincus, he developed what is now called Pincus theory \cite{Pincus1976,Pincus1977}, which consider the pulled polymer as a sequence of independent ``blobs''.  Brochard-Wyart further developed the ``trumpet'' and the ``stem-flower'' pictures of pulled polymer chain \cite{Brochard-Wyart1993a,Brochard-Wyart1994a,Brochard-Wyart1995,Adjarei1995a}. When the pulling force is not too strong, the polymer presents as a series of independent blobs with increasing size, i.e. the portion of polymer near to the free end fluctuates more. As the pulling force increased to a strong regime, the polymer portion near to fixed end is totally stretched, forming a ``stem-flower'' like picture, see in Fig. \ref{fig:stemFlower}.  
Using fluorescence microscope and optical tweezers,  Perkins et al. performed the pulling experiments on single DNA molecule and found the results consistent with the Brochard-Wyart's theory \cite{Perkins1995a,Perkins1994}. They also measured the relaxation time of this pulled polymer and obtained the scaling $\tau \sim L^{1.66}$ \cite{Perkins1994a}. Wirtz also confirmed the theory by measuring transport properties of a single DNA molecule in \cite{Wirtz1995}.

\begin{figure}[htpb]
    \centering
    \includegraphics[width=0.8\linewidth]{stemFlower}
    \caption{Illustration of ``stem-flower'' picture for pulled polymer chain. (a) Trumpet picture at moderate pulling force; (b) Stem-flower picture at strong pulling force. Image reprinted from \cite{Brochard-Wyart1995}.}
    \label{fig:stemFlower}
\end{figure}


Rezehak et al. considered pinned polymer in a uniform flow with hydrodynamic interaction and proposed a so-called f-shell blob model \cite{Rzehak1999}. Larson et al. performed Brownian Dynamics simulation for a DNA in an external flow field \cite{Larson1999}. In 2000, Doyle measured the cyclic and stretching dynamics of a tethered DNA molecule in the shear flow \cite{Doyle2000, Ladoux2000a}. Sebastian studied the dynamics of pulling a polymer out of a potential well \cite{Sebastian2000}. Cui performed the stretching and releasing experiment by pulling a single chicken erythrocyte chromatin fiber with the optical tweezers \cite{Cui2000a}. Rzehak discussed the conformation fluctuation and relaxation of a tethered polymer in uniform flow \cite{Rzehak2003a}. In 2007, Mohan et al. employed Rouse theory to study the unraveling dynamics of tethered semiflexible polymer in uniform solvent flow \cite{Mohan2007}. Sing et al. studied flexible and semiflexible tethered polymers in the limit of high shear flows and consequently near-full extension of the chains in \cite{Sing2011}.  Sakaue et al. studied the conformation and dynamics of a single flexible polymer chain that is pulled by a constant force applied at its one end, finite extensibility, the excluded volume, and the hydrodynamic interactions are discussed \cite{Sakaue2012a}.  Varghese et al. investigated the force fluctuations in stretching a tethered polymer \cite{Varghese2013}.
In 2013, Dai and Doyle found in \cite{Dai2013a} that the scenario of a pulled polymer is very similar to a polymer confined in a cylinder with proper radius.

To the best of our knowledge, the discussion of pulled ring polymer is missing except our own work \cite{Lin2015}. So we reviewed here the works about ring polymer on one side and the works of pulled polymer on the other side. Of course, the details are not shown here, interested readers can go into the references.

%********************************** %Third Section  *************************************
\section{Outline}  %Section - 1.3
\label{sec:outline}
With the introduction of the biological problem and the background of polymer modeling in previous two sections, we are ready to go into the details of our study. There are several specific models and techniques used in this thesis, such as the ASEP model. In order not to break the logic of the thesis, we will leave the brief background introduction of those right before the using. In this section, we will propose our research goals for this thesis and give an overview for the organisation of the thesis.

\subsection{Research goals}
\label{sub:research_goal}

The research goals of this thesis is listed as following:

$\bullet$ Propose a polymer model to describe the chromosomes in fission yeast during nuclear oscillation. This is actually already mentioned in the previous section, i.e. the pulled polymer loop model. However, the details of the model will be discussed in the following chapters.

$\bullet$ Develop the quantitative theory for our pulled looping polymer model. As far as we know, there is no previous work on this issue. On the other hand, it is shown by the experimental facts that this model is the best one for our problem. 

$\bullet$ Perform realistic numerical simulations of the nuclear oscillation of fission yeast using the polymer model. The theory is always simplified and has a lot of assumptions in order to analytically tractable. It is necessary to do a realistic simulation that can verify our theory on one hand, and comparable to the experimental data on the other hand.

$\bullet$ Using the physical insights to understand to biological processes such as chromosome alignment. Understand the biology is our ultimate goal. The chromosome movements during cell division are so important that many diseases are related to that, such as Down syndrome \cite{Patterson2009}. Our understanding from physical layer helps to fight with these diseases.

In one sentence, we want to quantitatively model the chromosome dynamics during the nuclear oscillation and use it to understand the biology. 

\subsection{Overview of the thesis}
\label{sub:organisation_of_the_thesis}

Add afterward.







